src/transformers/configuration_fsmt.py

```diff
@@ -0,0 +1,213 @@
+# coding=utf-8
+# Copyright 2019-present, Facebook, Inc and the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" FSMT configuration """
+
+
+import copy
+import logging
+
+from .configuration_utils import PretrainedConfig
+from .file_utils import add_start_docstrings_to_callable
+
+
+logger = logging.getLogger(__name__)
+
+FSMT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "stas/wmt19-ru-en": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-ru-en/config.json",
+    "stas/wmt19-en-ru": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-en-ru/config.json",
+    "stas/wmt19-de-en": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-de-en/config.json",
+    "stas/wmt19-en-de": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-en-de/config.json",
```

Comment:
Shouldn't we update these to `facebook/` instead of `stas/` @julien-c, given they're ports of weights originally pre-trained by the Fairseq team?

Reply:
Yeah. I don't this stas is allowed to write to that dir.

Reply:
Ah yes sounds good! Thanks!

Reply:
That has been the plan all along.

Reply:
We also have another set of 5 models ready to be merged for allen-nlp.

Let me know how to proceed.

Reply:
Done!